presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -228,6 +228,15 @@ Statistical Aggregate Functions
 
     Returns the sample covariance of input values.
 
+.. function:: kurtosis(x) -> double
+
+    Returns the (excess) kurtosis of all input values. Unbiased estimate using
+    the following expression:
+
+    .. code-block:: none
+
+        kurtosis(x) = n(n+1)/((n -1)(n - 2)(n-3)) sum[(x_i - mean)^4] / stddev(x)^4 - 3(n-1)^2/((n-2)(n-3))
```

Comment:
Minor nit, but would be nice to consistently put spaces around the minus sign or consistently not use spaces.